Update artifacts.md (#10706)
PrefectHQ/prefect: docs/concepts/artifacts.md

---
# Artifacts

> Artifacts are persisted outputs that make it easy to track and monitor the objects that your flows produce and update over time.  They can be published via the Prefect SDK or REST API.  They are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI.  Prefect currently supports three kinds of Artifacts: tabular data, markdown text, and web links.

Reviewer: Hi @odoublewen, thanks for taking this up! I do think we want to keep the same structure as before so that it's very clear what they're useful for.

Reviewer: I just committed a suggestion-- hope that's ok! thanks again!

Reviewer: haha no worries, appreciate it!

> Artifacts are persisted outputs that make it easy to track and monitor the objects that your flows produce and update over time.  They can be published via the Prefect SDK or REST API.  They are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI.  Prefect currently supports three kinds of Artifacts: tabular data, markdown text, and web links.

Reviewer: ```suggestion
Artifacts are persisted outputs such as tables, markdown, or links. They can be published via the Prefect SDK or REST API. They are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI. Artifacts make it easy to track and monitor the objects that your flows produce and update over time.
```
